source/Octostache/Templates/BuiltInFunctions.cs

```diff
@@ -67,8 +67,9 @@ static string Markdown(string raw)
             var options = new MarkdownOptions();
             options.AutoHyperlink = true;
             options.LinkEmails = true;
+            options.AllowEmptyLinkText = true;
 
-            return new Markdown(options).Transform(raw.Trim());
+            return new Markdown(options).Transform(raw.Trim()) + "\n";
```

Comment:
Nope - its a difference between `MarkdownSharp` and `Heyred.MarkdownSharp`.
Specifically:
<img width="1282" alt="screen shot 2016-08-10 at 10 24 52" src="https://cloud.githubusercontent.com/assets/373389/17538011/c1317f58-5ee4-11e6-951e-d2a7e2c7ea5e.png">
`MarkdownSharp` appends a "\n", whereas `HeyRed.MarkdownSharp` doesn't.

I'm tempted to pull in the unit tests for `MarkdownSharp` to ensure we get like for like behaviour with existing.